accounts/abi/bind/backends/simulated.go

```diff
@@ -41,6 +41,7 @@ import (
 var _ bind.ContractBackend = (*SimulatedBackend)(nil)
 
 var errBlockNumberUnsupported = errors.New("SimulatedBackend cannot access blocks other than the latest block")
+var errGasEstimationFailed = errors.New("Gas required exceeds block limit or always failing transaction")
```

Comment:
Please make 'Gas' is the message lowercase. All errors should be lowercase to allow composing them (the error above this is the exception as it refers to a public type)